types/google-apps-script/google-apps-script.document.d.ts

```diff
@@ -1462,6 +1465,53 @@ declare namespace GoogleAppsScript {
      *     table.getCell(0, 2).setVerticalAlignment(DocumentApp.VerticalAlignment.BOTTOM);
      */
     enum VerticalAlignment { BOTTOM, CENTER, TOP }
+
+    interface Date extends Element {
+      copy(): Date;
+      getAttributes(): any;
+      getDisplayText(): string;
+      getLocale(): string;
+      getNextSibling(): Element;
+      getParent(): ContainerElement;
+      getPreviousSibling(): Element;
+      getTimestamp(): Date;
+      getType(): ElementType;
+      isAtDocumentEnd(): boolean;
+      merge(): Date;
+      removeFromParent(): Date;
+      setAttributes(attributes: any): Date;
```

Comment:
It is better to define type of arg attributes.

```suggestion
      setAttributes(attributes: {[key in DocumentApp.Attribute]: any}): Date;
```

```ts
const foo = (attributes: {[key in Attribute]: any}) => {
    attributes[DocumentApp.Attribute.BACKGROUND_COLOR] = 1; // ok
    attributes["bar"] = 1; // compile error
}
```

Reply:
Hmmm, there are lots of other uses of attribute spread throughout the rest of this typing and they all use `any`.  Shouldn't they be consistent?  Someone could switch all of them, but I don't know enough to be confident that switching them all wouldn't break existing code.  Even if I did, I'd still opt to make that change in a separate PR.

Reply:
Okay, leave this issue as it is. I welcome if you make a another PR to refactor it, but it is not required 😃